loky/_base.py

```diff
@@ -16,20 +16,24 @@
 import threading
 import collections
 
-
+HAS_CONCURRENT_FUTURES = False
 if sys.version_info[:2] >= (3, 3):
```

Comment:
I would rather not merge this PR while we are in the process or dropping support for Python <3.7.